### Chess move status message.
Header        header    # standard ROS header
ChessMove     move      # chess move
